src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendars-grid">
+      <div class="calendar calendar--start-mon calendar--month-length-31">
```

Comment:
specify modifiers in this way (checklist p2): 
`block-name`--`modifier-name`--`modifier-value`


```suggestion
      <div class="calendar calendar--start-day--mon calendar--month-length--31">
```